.github/workflows/pullrequest.yml

```
@@ -80,11 +80,7 @@ jobs:
       - name: Calculate cache key for node modules
         id: calculate_node_modules_hash
         run: |
-          PACKAGE_JSON_HASH=$(cat package.json | jq '{resolutions,dependencies,devDependencies}' | sha1sum -t | cut -f1 -d" ")
-          echo "PACKAGE_JSON_HASH: $PACKAGE_JSON_HASH"
-          export NODE_MODULES_HASH=${{ runner.os }}-${{ hashFiles('yarn.lock') }}-$PACKAGE_JSON_HASH
-          echo "NODE_MODULES_HASH: $NODE_MODULES_HASH"
-          echo "node-modules-hash=$NODE_MODULES_HASH" >> $GITHUB_OUTPUT
+          echo "node-modules-hash=$(./scripts/ci/get-node-modules-hash.mjs)" >> $GITHUB_OUTPUT
```

could you add a debug below so we see the hash?

```suggestion
          HASH="$(./scripts/ci/get-node-modules-hash.mjs)"
          echo "node-modules-hash: ${HASH}"
          echo "node-modules-hash=${HASH}" >> $GITHUB_OUTPUT
```